pkg/cloud/vsphere/constants/constants.go

```diff
@@ -39,7 +39,7 @@ const (
 
 	// ReadyAnnotationLabel is the annotation used to indicate a machine and/or
 	// cluster are ready.
-	ReadyAnnotationLabel = "capv.sigs.k8s.io/ready"
+	ReadyAnnotationLabel = "capi.sigs.k8s.io/ready"
```

Comment:
Looks like this should live in CAPI eventually, cc @ncdc

Reply:
We should be careful here, if we move this to CAPI but rename the annotation, we now have a breaking change. OKAY for alpha I guess but something to consider.

Reply:
I'll rename this to be specific to CAPV.